src/ape/pytest/fixtures.py

```diff
@@ -77,11 +77,13 @@ def project(self) -> ProjectManager:
         return self.project_manager
 
     @pytest.fixture(scope="session")
-    def Contract(self):
+    def Contract(self, address: AddressType) -> ContractInstance:
```

Comment:
parameters to fixtures are other fixtures, meaning this would be referencing an `address` fixture? i don't think this is the right idea.